pjmedia/include/pjmedia/config.h

```diff
@@ -44,6 +44,34 @@
 #   include <pjmedia/config_auto.h>
 #endif
 
+/**
+ * Initial memory block for media endpoint.
+ */
+#ifndef PJMEDIA_POOL_LEN_ENDPT
+#   define PJMEDIA_POOL_LEN_ENDPT		512
+#endif
+
+/**
+ * Memory increment for media endpoint.
+ */
+#ifndef PJMEDIA_POOL_INC_ENDPT
+#   define PJMEDIA_POOL_INC_ENDPT		512
+#endif
+
+/**
+ * Initial memory block for event manager.
+ */
+#ifndef PJMEDIA_POOL_EVTMGR_LEN
+#   define PJMEDIA_POOL_EVTMGR_LEN		500
+#endif
+
+/**
+ * Memory increment for evnt manager.
+ */
+#ifndef PJMEDIA_POOL_EVTMGR_INC
+#   define PJMEDIA_POOL_EVTMGR_INC		500
+#endif
+
```

Comment:
Sorry just got a chance to revisit this PR.

IIRC we tried to reproduce the memory fragmentation issue (can't remember the result, can't find any doc/mail history either :D ). Anyway, some pools have been configurable already, so I think it is okay to integrate the patch, just perhaps better to try to maintain a consistent config name pattern, which mostly seem to use: `[MODULE]_POOL_LEN/INC_[OBJECT]`, e.g: `PJSIP_POOL_LEN/INC_TRANSPORT`.

Reply:
Hello there,

I tried to adhere to the name patterns I found the in modules in which I made the changes; but I'll go and revisit the changes to see if I missed something.

As for the fragmentation issue:
I specifically made only the pools configurable which I found to be problematic in my case.
I experimented with cleaning up pools under certain conditions (which definitely helped with the fragmentation issue)
but I wasn't happy with what I came up and stayed with the mitigation enabled by the changes proposed in this PR.